src/test/pipenv-test-app/Pipfile.lock

```diff
@@ -0,0 +1,385 @@
+{% if spec.version == "3.6" %}
```

Comment:
Wouldn't this be better to separate these into different files? That way if you want to regenerate it, you just overwrite the file instead of having to copy paste contents into this big jumble.

Reply:
It probably would be better, unfortunately design of manifest.sh https://github.com/mcyprian/s2i-python-container/blob/ultimate-distgen/manifest.sh#L29 file doesn't allow to have different source files for different versions. The whole idea of manifest and generate rule is to generate everything from one collection of source files/templates.

Reply:
As we talked, the different files could be included from this one.

Reply:
Fixed.